lib/metrics/metrics.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
+// Licensed under the Apache License, Version 2.0. See LICENSE
+
+export default {
+  TTFCP: 'first-contentful-paint',
+  TTFMP: 'first-meaningful-paint',
+  TTF_CPU_IDLE: 'first-cpu-idle',
+  TTCI: 'interactive',
+  PSI: 'speed-index',
+  VISUALLY_COMPLETE: 'vc100',
```

Comment:
seems like this isn't being reported. 

While it's *possible* to get the number, you'll have to choose if you want to go with devtools throttling just to get this.

Reply:
yup, just dummy placed here becase of confusion with vizComplete. 
wdyt if we call it just like it's done in LH to not involve extra effort into `you'll have to choose if you want to go with devtools throttling just to get this.` ?

Reply:
sure, but right now i think there's no configuration option for pwmetrics to use devtools throttling instead of the default simulation. that'll be changes to your config. 

might want to delay this and do it later, since it adds some complexity.

Reply:
ok, dropping it then.
will you be ok with just 5 metrics?